src/standup/standup.controller.ts

```diff
@@ -23,24 +24,31 @@ export class StandupController {
   @Post()
   @UseGuards(TokenGuard)
   @ApiSecurity('token')
-  createStandup(@Body() body: StandupDTO) {
-    return this.standupService.create(body);
+  @ApiHeader({ name: 'keyspace', required: true })
+  createStandup(
+    @Body() body: StandupDTO,
+    @Headers('keyspace') keyspace: string,
+  ) {
+    return this.standupService.create(body, keyspace);
```

Comment:
The provided Keyspace needs to be validated, as well as protected. So users cant get data which belong to a different server

Reply:
Oh yes 👍 , it will be provided by the Discord Bot that we trust because it will have the authorised token. But we had everything for read (GETs) open which might be an issue